noxfile.py

```diff
@@ -180,3 +181,64 @@ def make(session: nox.Session):
         *(args.make_args or ("clean", "coredocs")),
     ]
     session.run("make", "-C", "docs/docsite", *make_args, external=True)
+
+
+def _rm_tmpdir(tmpdir):
+    rmtree(tmpdir, ignore_errors=True)
+
+
+@nox.session
+def tag(session: nox.Session):
+    """
+    Check the core repo for new releases and create tags in ansible-documentation
+    """
+    install(session, req="tag")
+
+    CORE_REPO = "https://github.com/ansible/ansible"
+    DOCS_REPO = "https://github.com/ansible/ansible-documentation"
+
+    args = list(session.posargs)
+    tmpdir = None
+
+    if not any(arg.startswith(("--core", "--docs")) for arg in args):
+        if not which("git"):
+            session.error("git is not installed or not found int PATH")
```

Comment:
This is redundant. You can't run the noxfile without a git checkout to begin with.

Reply:
You can download a zip from GitHub which would contain it. 

Also, this ensures that when/where nox is running, it has `git` accessible.